samples/matter/common/src/bridge/ble_connectivity_manager.h

```diff
@@ -182,6 +186,23 @@ class BLEConnectivityManager {
 	static void DiscoveryCompletedHandler(bt_gatt_dm *dm, void *context);
 	static void DiscoveryNotFound(bt_conn *conn, void *context);
 	static void DiscoveryError(bt_conn *conn, int err, void *context);
+	static int StartGattDiscovery(bt_conn *conn, BLEBridgedDeviceProvider *provider);
+
+#ifdef CONFIG_BT_SMP
+	static void SecurityChangedHandler(struct bt_conn *conn, bt_security_t level, enum bt_security_err err);
+	static void AuthenticationCancel(struct bt_conn *conn);
+	static void PasskeyEntry(struct bt_conn *conn);
+	static void PairingComplete(struct bt_conn *conn, bool bonded);
+	static void PairingFailed(struct bt_conn *conn, enum bt_security_err reason);
+
+	/**
+	 * @brief Set authentication pincode to confirm the specific Bluetooth LE connection.
```

Comment:
Why only this one is documented? If the rest of the methods guarded with CONFIG_BT_SMP are used only internally we should move them to the private scope of the class.

Reply:
Yes, these are only static handlers used internally and cannot be called by the user directly. Sure, I added it to public, similarly to other currently existing handlers, but I agree all of them could be moved to the private scope. I will do that.

Reply:
As discussed F2F, it turned out that they have to be public. Added comment to the methods declaration in the class.